packages/language-support/src/autocompletion/completionCoreCompletions.ts

```diff
@@ -665,11 +666,20 @@ export function completionCoreCompletion(
       }
 
       if (ruleNumber === CypherParser.RULE_symbolicAliasName) {
-        return completeAliasName({ candidateRule, dbSchema, parsingResult });
+        return completeAliasName({
+          candidateRule,
+          dbSchema,
+          parsingResult,
+        });
```

Comment:
I don't know if the grammar changed (possibly) but this looks like it can never be a parameter